### PR TITLE
Update Swagger Specifications. 

### PR DESCRIPTION
**Pull request by cnulenka**
New Api endpoints information added to be used for Clair Integration.

### CODE REVIEW

**Comment by cnulenka** [comment]
[WIP] Have added the initial changes to PR. For creating public facing api endpoints .

File: bayesian/api_v1.py [hunk 1]
```diff
@@ -1205,6 +1205,21 @@ def get(ecosystem):
         return jsonify(result)
 
 
+class CvesByDateEcosystem(ResourceWithSchema):
```
Comment (miteshvp) [inline bayesian/api_v1.py]: You should not create this class in api_v1.py file. Please create a new file under `bayesian` directory

File: bayesian/utils.py [hunk 2]
```diff
@@ -172,20 +172,19 @@ def __init__(self, package, version, response):
         self._package = package
         self._version = version
         self._cves = []
-        self._nocve_versions = []
+        # Gremlin query will always return two elements for response
```
Comment (miteshvp) [inline bayesian/utils.py]: It seems all the changes are being pulled in from my PR. You need to rebase your branch after pulling latest changes.

**Comment by miteshvp** [comment]
Seems lots of commits are being pulled from other PRs. Please rebase your branch and then push the changes that are really relevant for this PR. Else, request you to close this one and create a new PR after rebasing your master branch.

**Pull request closed without merging**